exchange/bittrex.go

```
@@ -402,7 +413,18 @@ func (self *Bittrex) FetchTradeHistory() {
 			}
 			wait.Wait()
 			data.Range(func(key, value interface{}) bool {
-				result[key.(common.TokenPairID)] = value.([]common.TradeHistory)
+				tokenPairID, ok := key.(common.TokenPairID)
+				//if there is conversion error, continue to next key,val
```

is it OK to silently storeing incomplete trade history? Or it is better to return an error and try again next time? Please recheck other places with this same business logic.

---

After carefully rethink the matter, the logic is changed as follow: if the data returned from exchange is corrupted, skip this itter and wait till the next.